accelerator/help.h

```diff
@@ -0,0 +1,54 @@
+#ifndef ACCELERATOR_HELP_H_
+#define ACCELERATOR_HELP_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define SEE_MORE_LINK "https://github.com/DLTcollab/tangle-accelerator"
+
+typedef enum ta_cli_arg_value_e {
+  TA_HELP = 100,
+
+  /** TA */
+  TA_HOST,
+  TA_PORT,
+  TA_THREAD_COUNT,
+  TA_VERSION,
+
+  /** IRI */
+  IRI_HOST,
+  IRI_PORT,
+
+  /** REDIS */
+  REDIS_HOST,
+  REDIS_PORT,
+
+  /** CONFIG */
+  MILESTONE_DEPTH,
+  MWM,
+  SEED,
+} ta_cli_arg_value_t;
+
+typedef enum ta_cli_arg_requirement_e {
+  NO_ARG,
+  REQUIRED_ARG,
+  OPTIONAL_ARG
+} ta_cli_arg_requirement_t;
+
+static struct ta_cli_argument_s {
+  char* name;
+  int val;
+  char* desc;
+  ta_cli_arg_requirement_t has_arg;
+} ta_cli_arguments_g[] = {
+    {"ta-help", TA_HELP, "Show tangle-accelerator usage."}};
```

Comment:
`NO_ARG` here.

Reply:
I think `ta-help` can enable TA has higher compatibility to the entangled commands (if needed). 
Will we call entangled commands in the future?

Reply:
No, we won't use any of the ciri commands.

Reply:
No, we won't need to use ciri commands.